File: C#/344-reverse-string.cs
public class Solution {
    public string ReverseString(string s) {
        char[] sRaw = s.ToCharArray();
        StringBuilder builder = new StringBuilder();
        for(int i = sRaw.Length - 1; i >= 0; i--) {
            builder.Append(sRaw[i]);
        }
        return builder.ToString();
    }
}

File: Java/344-reverse-string.java
class Solution {
    // Time: O(n), Space: O(n)
    // Explanation: store string in character array, swap values from front and end while moving towards center of array. Return swapped char array as a string.
    public String reverseString(String s) {
        char[] charArr = s.toCharArray();
        int begin = 0, end = charArr.length - 1;
        while(begin <= end) {
            char temp = charArr[begin];
            charArr[begin] = charArr[end];
            charArr[end] = temp;
            begin++;
            end--;
        }
        return new String(charArr);
    }
}